tests/integration/conftest.py

```diff
@@ -57,36 +57,37 @@ def get_report_files():
 def pytest_collection_modifyitems(session, config, items):
     selected_tests = []
     deselected_tests = []
+    if not global_parameters.avoid_platform_based_deselection:
+        for item in items:
+            supported_platforms = PLATFORMS.intersection(mark.name for mark in item.iter_markers())
+            plat = sys.platform
```

Comment:
Why `plat` and not `platform`?

Reply:
This is not related to the development. Changes only are aimed to ignore the platform deselection in case of using the defined variable